DirectEmployers/MyJobs
# [PD-3847] Add Delete to Partner Assessments

## What's this PR do?
Adds delete functionality to partner assessments

## Where should the reviewer start?
Partner Assessments

## How should this be manually tested?
Create an assessment
Edit that assessment
Click "delete"
Confirm
See that it doesn't show up in the list (it's actually archived, not deleted)

## Any background context you want to provide?
This relies on the ubuntu:xenial changes in another PR. 

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
